.github/ISSUE_TEMPLATE/api-suggestion.md

```diff
@@ -0,0 +1,54 @@
+---
+name: API proposal
+about: Propose a change to the public API surface
+title: ''
+labels: [Feature Request, Concept-API]
```

Comment:
Note: Sometimes people open issues using the wrong template, leading to wrong labels applied. i.e, they open the API suggestion template, empty it out, and then write an issue unrelated to an API suggestion. I have a pending-review PR to remove the labeling from another issue template (https://github.com/dotnet/roslyn/pull/53155). However, if you think it's not a problem to have incorrect labels initially that later on you'll correct, then keep it.

Reply:
I don't think it's a problem, and I'd rather have these applied than need to go and apply them to every new proposal.